vllm-tensorizer/Dockerfile

```
@@ -24,11 +29,13 @@ RUN export \
       ninja-build && \
     apt-get clean
 
-RUN apt update && \
-    apt install -y python3-pip python3-packaging \
-      git ninja-build && \
-    pip3 install -U --no-cache-dir pip packaging setuptools wheel
+RUN ldconfig
 
+RUN apt-get -qq update && \
+    apt-get -qq install -y --no-install-recommends \
+      python3-pip python3-packaging git ninja-build && \
```

The `apt` package `python3-packaging` is equivalent to `pip3 install -U packaging` below

```suggestion
      python3-pip git ninja-build && \
```